finance-api/user-guides/fetch-account-balance.md

```diff
@@ -0,0 +1,50 @@
+Fetch account balance
+===
+Use the Finance API to fetch the account balance of the merchant's preliminary or liquid account.
+
+* [Prerequisites](#prerequisites)
+* [Fetch the account balance](#fetch-the-account-balance)
+* [Related task](#related-task)
+* [Related API reference](#related-api-reference)
+
+## Prerequisites
+* Make sure that authorization is set up with the required OAuth scope using [Authorization OAuth2 API](../../authorization.adoc).  
+<!-- to be continued if any -->
+
+## Fetch the account balance 
+Fetch the balance in one or both of the merchant's preliminary and liquid accounts as follows:
+   * To check the balance of transactions that are still being cleared by the acquiring bank, fetch the preliminary account balance:
```

Comment:
This is not correct. Transactions that are in the preliminary is because they weren't liquefied yet as we wrote in the documentation. The transaction goes from preliminary to liquid when:

```
Zettle checks whether the payment is ready to be paid out to the merchant.
```

So the correct here would be something like
```
To check the balance of transactions that are still being checked by Zettle whether they should be paid out or not, fetch the preliminary account balance:
```

Then next step is correct, which says that liquid balance is what is ready to be paid out.